### PR TITLE
Feature/baomt/t01 show the total number of jobs

### PR DESCRIPTION
**Pull request by tminhbao**

### CODE REVIEW

**Review by TruongNP — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 1]
```diff
@@ -25,6 +25,23 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <link
```
[the remark this answers is not in the document]
Comment (tminhbao) [reply public/index.html]: Fixed

File: src/components/Jobs/JobCard/JobCard.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.salary,
```
[the remark this answers is not in the document]
Comment (tminhbao) [reply src/components/Jobs/JobCard/JobCard.scss]: Fixed

File: src/components/Jobs/JobCard/JobCard.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+type Props = {
+  item: any;
+};
+
+const JobCard = (props: Props) => {
+  const { item } = props;
+  return (
+    <div
+      className="container p-2 my-2 overflow-hidden"
+      style={{
```
[the remark this answers is not in the document]
Comment (tminhbao) [reply src/components/Jobs/JobCard/JobCard.tsx]: Fixed

File: src/components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./Navbar.scss";
+
+type Props = {};
+
+const Navbar = (props: Props) => {
+  return (
+    <div className="container">
```
[the remark this answers is not in the document]
Comment (tminhbao) [reply src/components/Navbar/Navbar.tsx:9]: Fixed

File: src/components/SearchBar/SearchBar.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  return (
+    <div className="bg-dark container-fluid" style={{ padding: "80px 200px" }}>
+      <div className="row">
+        <h1 className="text-white" style={{ letterSpacing: "-2px" }}>
+          {jobList.length} IT Jobs For Developers
+        </h1>
+      </div>
+      <div className="row">
+        <div className="col-3">
+          <select className="form-select w-100" defaultValue={1}>
```
[the remark this answers is not in the document]
Comment (tminhbao) [reply src/components/SearchBar/SearchBar.tsx:35]: Fixed

File: yarn.lock [hunk 6]
```diff
@@ -1172,6 +1172,13 @@
   dependencies:
```
[the remark this answers is not in the document]
Comment (tminhbao) [reply yarn.lock:1172]: Fixed

**Review by TruongNP — APPROVED** [review]
This PR has been approved

**Pull request merged**